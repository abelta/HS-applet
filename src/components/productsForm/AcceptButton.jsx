import React from 'react';
import PropTypes from 'prop-types';

const AcceptButton = ({ onClick }) => <button onClick={onClick}>Save</button>;

AcceptButton.propTypes = {
  onClick: PropTypes.func.isRequired,
};

export default AcceptButton;
