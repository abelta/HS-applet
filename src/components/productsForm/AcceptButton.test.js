import React from 'react';
import renderer from 'react-test-renderer';
import AcceptButton from './AcceptButton';

describe('AcceptButton', () => {
  const tree = renderer.create(<AcceptButton onClick={jest.fn} />).toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
