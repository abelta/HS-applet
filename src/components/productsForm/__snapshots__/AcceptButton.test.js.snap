// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AcceptButton renders correctly 1`] = `
<button
  className="btn-flat waves-effect waves-light green white-text"
  disabled={false}
  onClick={[Function]}
  style={
    Object {
      "borderRadius": 0,
    }
  }
  type="submit"
>
  Save
  <i
    className="material-icons right"
  >
    send
  </i>
</button>
`;
